third_party/gtest-1.8.0/include/gtest/internal/gtest-port.h

```diff
@@ -689,6 +689,22 @@ struct _RTL_CRITICAL_SECTION;
 #  include "gtest/internal/gtest-tuple.h"  // IWYU pragma: export  // NOLINT
 # elif GTEST_ENV_HAS_STD_TUPLE_
 #  include <tuple>
+
+#if defined(J9ZOS390)
```

Comment:
Yes this is the only modification. There were previous modifications in this area as well in the current codebase [1] as part of #1429. It seems we did a CQ at the time because it was a large contribution. I don't suspect we need one here. I would however assume that there would be no licensing implications given the originally Google Test code was changed with the addition of [1] and that passed the CQ already.

[1] https://github.com/eclipse/omr/blob/7574eec990ab5d396bcec38b02a160f9381e8b55/third_party/gtest-1.8.0/include/gtest/internal/gtest-port.h#L2277-L2296

Reply:
I see. That answered my question. Thanks.